### PR TITLE
june timestamp implementation for time out

### PR DESCRIPTION
**Pull request by kevinzhangftw**

### CODE REVIEW

File: echoapp_client.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def ping(*args):
 if __name__ == "__main__":
     # websocket.enableTrace(True)
     parser = argparse.ArgumentParser(description='Arguments to start echoapp_client')
-    parser.add_argument('--host', type=str, default="ws://voiceminder.localtunnel.me/websocket/",
```
Comment (jyleong) [inline echoapp_client.py]: revert back to ws://voiceminder.localtunnel.me/websocket/

File: src/asynchronous/countdown.py [hunk 2]
```diff
@@ -20,3 +20,18 @@ def run(self):
...
+        self.method = method
+        self.state = state
+        self.countdownTimer = None
+
+    def run(self):
+        self.countdownTimer = threading.Timer(15, function=self.method, args=(self.state,))
+        self.countdownTimer.start()
+
+    @staticmethod
+    def stop():
```
Comment (jyleong) [inline src/asynchronous/countdown.py]: take out the @static decorator, and add self argument

File: src/user/User.py [hunk 3]
```diff
@@ -12,8 +12,25 @@ class User(object):
   name = None
   state = UserState.Nameless
   conversant = None
+  timestamp = None
   """docstring for User"""
   def __init__(self):
     super(User, self).__init__()
+
+  def setState(self, newState):
```
Comment (jyleong) [inline src/user/User.py]: this method can be simplified to:
self.state = newState

if newState is the enum and not the integer

**Comment by kevinzhangftw** [comment]
went with james request

**Pull request closed without merging**